src/fsharp/CompileOps.fsi

```diff
@@ -580,6 +580,7 @@ val IsOptimizationDataResource : ILResource -> bool
 
 /// Determine if an IL resource attached to an F# assemnly is an F# quotation data resource for reflected definitions
 val IsReflectedDefinitionsResource : ILResource -> bool
+val GetSignatureDataResourceName : ILResource -> string
```

Comment:
This is used by some functionality in FSharp.Compiler.Service so adding it to the signature file